.circleci/ds005_outputs.txt

```diff
@@ -49,6 +53,8 @@ fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-MNI152NLin20
 fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-MNI152NLin2009cAsym_variant-smoothAROMAnonaggr_preproc.nii.gz
 fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_AROMAnoiseICs.csv
 fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_confounds.tsv
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_label-aparcaseg_roi.nii.gz
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_label-aseg_roi.nii.gz
```

Comment:
@satra Do you want to have a look at these new proposed outputs when you get a second? Does this look like what you had in mind?

Reply:
@effigies - looks good to me.  and just to clarify all runs are realigned to each other during fmriprep processing or left in some self-realigned state?

Reply:
Satra's comment here made me realize we still need to specify the target space for the BOLD-sampled ROIs here with a `space-T1w` entity.

Reply:
@effigies - aren't the bold sampled rois in the functional space of whatever run was processed?

Reply:
The current name suggest that they are in the space of the corresponding functional scan, but @effigies will know for sure.